contracts/adapters/voting/OffchainVoting.sol

```diff
@@ -111,18 +115,18 @@ contract OffchainVotingContract is
         bytes32 nameHash;
         bytes32 bodyHash;
         string[] choices;
-        uint256 start;
-        uint256 end;
+        uint64 start;
+        uint64 end;
         string snapshot;
     }
 
     struct VoteMessage {
-        uint256 timestamp;
+        uint64 timestamp;
         VotePayload payload;
     }
 
     struct VotePayload {
-        uint256 choice;
+        uint32 choice;
         bytes32 proposalHash;
```

Comment:
Looking at the other code this should perhaps be `bytes32 proposalId`?

Reply:
Actually, it looks as if `VoteMessage` and `VotePayload` are unused in this file. It appears we are using the ones from `SnapshotProposalContract.sol`

Reply:
that's correct. Let me remove them